### PR TITLE
[#52] Support for ConceptAttribute.

### PR DESCRIPTION
**Pull request by samuelmale**
Issue https://github.com/mekomsolutions/openmrs-module-initializer/issues/52

### CODE REVIEW

File: api/src/test/resources/testdata/test-concepts.xml [hunk 1]
```diff
@@ -40,5 +40,7 @@
 
   <concept_set CONCEPT_SET_ID="1089" UUID="75e21f76-a09a-4c53-a401-13d2f38628b0" SORT_WEIGHT="1000.0" DATE_CREATED="2019-12-11 17:18:05.0" CONCEPT_ID="5501" CONCEPT_SET="5503" CREATOR="1"/>
   <concept_set CONCEPT_SET_ID="1090" UUID="c99b2737-c824-48f9-b095-c8cf516b9532" SORT_WEIGHT="1010.0" DATE_CREATED="2019-12-11 17:18:05.0" CONCEPT_ID="5502" CONCEPT_SET="5503" CREATOR="1"/>
-  
+  <concept_attribute_type concept_attribute_type_id="1" name="Audit Date" datatype="org.openmrs.customdatatype.datatype.DateDatatype" uuid="9516cc50-6f9f-11e0-8414-001e378eb67e" creator="1" date_created="2020-04-03 00:00:00.0" min_occurs="0" retired="false" />
+  <concept_attribute_type concept_attribute_type_id="2" name="Email Address" datatype="org.openmrs.customdatatype.datatype.FreeTextDatatype" uuid="k516cc50-6f9f-11e0-8414-101e378e467f" creator="1" date_created="2020-04-03 00:00:00.0" min_occurs="0" retired="false" />
+  <concept_attribute concept_attribute_id="1" concept_id="5500" attribute_type_id="1" value_reference="2011-04-25" uuid="3a2bdb18-6faa-11e0-8414-001e378eb67e" creator="1" date_created="2020-04-03 00:00:00.0" voided="false" />
```
Comment (mks-d) [inline api/src/test/resources/testdata/test-concepts.xml:45]: Don't use IDs 1, 2... etc, they will end up clashing with other datasets (such as the one from Core). Just do something like 3089, 3090... etc.

**Comment by samuelmale** [comment]
> Is it possible that `ConceptsCsvParserTest` fails?
> 
> This happened locally for me, but I just looked quickly:
> 
> ```
> process_shouldParseBaseCsv(org.openmrs.module.initializer.api.c.ConceptsCsvParserTest):
>   expected:<1> but was:<2>
> ```

Nice catch @mks-d, I wonder how I missed doing a: `mvn clean install` after working a minor followup commit.

File: api/src/test/resources/testdata/test-concepts.xml (hunk 1 above)
Comment (samuelmale) [reply api/src/test/resources/testdata/test-concepts.xml]: Resolved

**Comment by samuelmale** [comment]
@mks-d  I have updated the PR, though I wonder why Travis CI is failing. I gambled around to restart the travis build but failed.

**Pull request merged**